src/app/space/settings/areas/areas.component.html

```diff
@@ -1,47 +1,39 @@
+<areas-toolbar
+    (onAddArea)="addChildArea()"
+    (onFilterChange)="filterChange($event)"
+    (onSortChange)="sortChange($event)"
+    [resultsCount]="resultsCount">
+</areas-toolbar>
 <div class="container-fluid">
   <div class="row">
-    <div class="col-xs-8 col-sm-8 col-md-9">
-      <h3 class="areas-header">Areas that make up the {{context.space.attributes.name | spaceName}} Space:</h3>
-    </div>
-    <div class="col-xs-4 col-sm-4 col-md-3">
-      <div class="table-action-heading" (click)="addChildArea()">
-        <a><i class="pficon pficon-add-circle-o"></i> Add Areas</a>
-      </div>
-    </div>
-  </div>
-  <div class="row">
-    <div class="col-md-12">
-      <pfng-list
-        [config]="listConfig"
-        [itemTemplate]="itemTemplate"
-        [actionTemplate]="actionTemplate"
-        [items]="areas" >
-        <ng-template #itemTemplate let-item="item" let-index="index">
+    <div class="col-sm-12">
+      <pfng-tree-list *ngIf="showTree === true"
```

Comment:
Use `*ngIf="showTree"` rather than a comparison evaluation

Reply:
Actually, the `showTree` can be removed entirely as it isn't used after the change below.

Reply:
Sorry, but we cannot remove the timeout and achieve the desired functionality. 

This forces a change detection so that the existing expansion toggles can be rendered properly. If you remove the timeout, the area is still added, but the toggles don't function the way I described.

Reply:
Can you elaborate on 'existing expansion toggles can be rendered properly'?